test/ppx_import_support/test.ml

```diff
@@ -0,0 +1,102 @@
+(* Test for the ppx_import old syntax compat support *)
+
+open Ppxlib
+
+let id =
+  Extension.__declare_ppx_import
+    "id"
+    (fun ~ctxt:_ td ->
+       match td.ptype_manifest with
+       | Some {ptyp_desc = Ptyp_extension (_, PTyp wrapped_manifest); _} ->
+         {td with ptype_manifest = Some wrapped_manifest}
+       | _ -> assert false)
+[%%expect{|
+val id : Extension.t = <abstr>
+|}]
+
+Driver.register_transformation
+  ~rules:[Context_free.Rule.extension id]
+  "id"
+[%%expect{|
+- : unit = ()
+|}]
+
+(* The expander receives the type decl with the extension point removed, it should preserve
+   attibutes *)
+type t = [%id: int]
+[%%expect{|
+type t = int
+|}]
```

Comment:
Do you test here if it preserves attributes? Is there a way to test that with an expect test?

Reply:
The `@@deriving test below should be enough for that!

Reply:
Oh, I see. So the comment was a little bit misleading to me.